### ISSUE TITLE
Unable to install script (Windows)

### ISSUE THREAD

**Issue by Zer0FoxGiven** [OmegaPhil/kvirc-highlight-management-script#2]
Hello,

today I updated my KVIRC. The Info about KVIRC is as follows - 

_KVIrc 4.9.2 'Aria'

Runtime Info:
System name: Windows Release : 6.1.7601
System version: Windows 7 Ultimate Service Pack 1 (Build 7601)
Architecture: x86 Family 6 Model 60 Stepping 3
Qt version: 5.8.0
Qt theme: windowsvista

Build Info:
Build date: 2017-05-24 19:11:27 UTC
Sources date: 20170524
Revision number: git-7109-gae97a8ffa
System name: Windows-6.3.9600
CPU name: AMD64
Build command: C:/Program Files (x86)/CMake/bin/cmake.exe
Build flags: 
   MANDIR=share/man
   CMAKE_INSTALL_PREFIX=C:/projects/kvirc/build/release
   Threads=Win32
Compiler name: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64_x86/cl.exe
Compiler flags: /DWIN32 /D_WINDOWS /W3 /GR /EHsc
Qt version: 5.8.0
Features: IRC, IPv6, Crypt, SSL, IPC, Transparency, Perl, Python, Enchant, Qt5, KVS_

Then I tried installing HighlightManagement. Here are my observations -

1. The KVS provided isn't clickable, so I had to manually install it as per instructions provided.
2. There was no option 'Scripts' created.
3. Here is the output of the 'Debugging Window' when I tried to parse KVS script.

_[KVS] Compilation error: Found character '<' (Unicode 3c) where an instruction was expected
[KVS]   In script context "C:/Users/pseudonym/Desktop/HighlightManagementScript.kvs", line 7, near character 1
[KVS] Code listing:
[KVS]   5 ...
 [KVS]   6 
[KVS]   7 <!DOCTYPE html>
[KVS]   8 <html lang="en">
[KVS]   9 ...
[KVS] Window:
[KVS]   CONSOLE [ID: 1]
[KVS]_

**Comment by OmegaPhil** [comment]
Can you open the downloaded script in something like Notepad, does it look intact?

**Comment by Zer0FoxGiven** [comment]
Opened it in Notepad++, and the script looks as it should. No garbled text, nothing.

**Comment by OmegaPhil** [comment]
k, I'll need to take a look at it in a fresh KVIrc profile then. Could be a Windows problem but this stuff is supposed to be multiplatform...

**Comment by OmegaPhil** [comment]
For ticket: Might be something to do with new 'Options controller'?

**Comment by OmegaPhil** [comment]
Right, reached this now - I have installed the latest git KVIrc on a VM where I killed off the previous config (and install) - revision number git-7117-ge60a55155 - and I was able to successfully run the script installation and startup commands, with the exception of 'User-friendly commandline mode' being enabled by default, resulting in the following error:

```
[20:07:02] [KVS] Warning: Failed to load the file '/home/omega/"/home/omega/files/source_code/kvirc/Highlight' for parsing
[20:07:02] [KVS]   In script context "commandline::userfriendly", line 1, near character 1
```

I don't think this is what is affecting you, but regardless to kill off that mode, at the bottom right of the window, to the far right of the command entry widget, press the arrow button to expand, then press the 3rd button from the left - normal KVS command mode is then active (and the parse command works).

As for the issue in general, it needs more people to report it... which presumably means it'll never get any further than this :/ It might just be a Windows issue somehow.
